lib/ansible/plugins/callback/default.py

```diff
@@ -115,6 +115,9 @@ def v2_on_file_diff(self, result):
         if 'diff' in result._result and result._result['diff']:
             self._display.display(self._get_diff(result._result['diff']))
 
+    def _get_censored_item(self, result):
```

Comment:
Also would mean that it's impossible for callback plugins to be guaranteed access to the args/results at all- probably fine for logging plugins, but right or wrong, people *are* doing other sinister things with callback plugins that could be broken by in-place censoring.